.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ["bug"]
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear description what is currently happening 
+
+**Did it work earlier?**
+If this something which worked well some time ago, please let us know about version where it works or at date when it worked.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Version**
+ - OS: [e.g. Windows, MacOS intel, MacOS Arm, Android, Linux, iOS]
```

Comment:
```suggestion
 - OS: [e.g. Windows, macOS Intel, macOS ARM, Android, Linux, iOS]
```

Reply:
maybe turn it into a list of checkboxes?

Reply:
let's start from something simple. Later we can evolve it

Reply:
I don't think that we can add checkboxes into *templates*. Github has them in "issue forms", which I think is different from what is in this PR.

Reply:
you can, here's an example: https://github.com/xbmc/xbmc/blob/master/.github/ISSUE_TEMPLATE/bug_report.md

Reply:
Ah, forgot about those. Not sure, multi-select for OS looks a bit unusual